vmdb/app/models/miq_server.rb

```diff
@@ -513,7 +513,6 @@ def restart
     restart_script = File.join(Rails.root, "vmdb_restart")
     File.chmod(0755, restart_script)
 
-    cmd = "#{restart_script} 2>&1 >> #{logfile}"
```

Comment:
YOLO.  This was probably me.   I vaguely recall moving to using spawn with [:out, :err] .

:fire: